front/app/components/Sharing/SharingButtons/index.tsx

```diff
@@ -158,6 +163,19 @@ const Buttons = styled.div`
         background: ${(props: any) => darken(0.1, props.theme.colorMain)};
       }
     }
+
+    &.copylink {
```

Comment:
Could we somehow use our IconButton from the component library and get rid of all of these things? (styled icons, Buttons styled div, etc.)?

Reply:
If too much effort - also fine to leave it as is!